src/main/java/io/confluent/idesidecar/restapi/messageviewer/data/SimpleConsumeMultiPartitionRequest.java

```diff
@@ -20,4 +20,41 @@ public record PartitionOffset(
       @JsonProperty("partition_id") int partitionId,
       @JsonProperty("offset") long offset
   ) {}
+
+  public SimpleConsumeMultiPartitionRequest withMaxPollRecords(Integer maxPollRecords) {
+    return new SimpleConsumeMultiPartitionRequest(
+        partitionOffsets, maxPollRecords, timestamp, fetchMaxBytes, messageMaxBytes, fromBeginning
+    );
+  }
```

Comment:
Can we use `@RecordBuilder` here to eliminate the need for these method implementations?